lib/rouge/lexers/csound/builtins.rb

```diff
@@ -0,0 +1,1639 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class CsoundOrchestra
+      # Opcodes in Csound 6.09.0 at commit 73a8df5 from
+      #   csound --list-opcodes3
```

Comment:
Not really. Csound outputs a list of _installed_ opcodes, and installing them all can involve building them from source, which is often not straightforward. (This file includes opcodes that I don’t have installed and don’t know how to install.)